.github/pr_title_checker_config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "LABEL": {
+        "name": "title needs formatting",
+        "color": "EEEEEE"
+    },
+    "CHECKS": {
+        "regexp": "^\\[[^]]*\\]* ",
```

Comment:
Will this break on automatically generated revert PRs?  Do we want it to allow those?

Reply:
good point... expanded it somewhat